### DISCUSSION TITLE
Chunk options value defined as code

### DISCUSSION THREAD

**Discussion by statquant** [quarto-dev/quarto-cli#9902] [Feature Requests]
Hello, 
In R using the knitr engine, it is possible to specify chunk options (`echo`, `eval`, ...) to be evaluated as code. My humble opinion is that this is a killer feature that is not available anywhere else.
I was wondering if there was any project to port this feature to be available using a jupyter engine ?

PS: please kindly let me know if this is not the place to be asking non-technical question of the sort

**Comment by cscheid** [comment]
I hear you: this is a popular request.

Unfortunately, we're unlikely to ever add general support for this because Quarto is, by design, agnostic to the language being executed, and these `!expr` tricks in knitr, powerful as they may be, make it impossible for Quarto to provide YAML intelligence (validation, completion, etc) in a way that works across engines and programming languages.

**Reply by statquant** [reply]
Thanks @cscheid for this prompt response!
We'll still have knitr 😩
